src/marqo/core/embed/embed.py

```diff
@@ -67,6 +76,14 @@ def embed_content(
         else:
             raise base_exceptions.InternalError(f"Content type {type(content)} is not supported for embed endpoint.")
 
+        # Decide on the prefix
+        if content_type == "query":
+            prefix = determine_text_prefix(None, marqo_index, "text_query_prefix")
+        elif content_type == "document":
+            prefix = determine_text_prefix(None, marqo_index, "text_chunk_prefix")
+        else:
+            prefix = determine_text_prefix(content_type, marqo_index, "text_query_prefix")
```

Comment:
There should be an option to have no prefix at all. Maybe when `content_type` is `None`